gpMgmt/test/behave/mgmt_utils/gpperfmon.feature

```diff
@@ -101,6 +101,32 @@ Feature: gpperfmon
         """
         Then wait until the results from boolean sql "SELECT count(*) > 0 FROM queries_history WHERE query_text = 'SELECT pg_sleep(80)'" is "true"
 
+    @gpperfmon_query_history
+    Scenario Outline: gpperfmon does not log nested statements with log_min_messages < debug4
```

Comment:
maybe `>`?

Reply:
Maybe even "DEBUG4 or DEBUG5", the ordering of these constants is confusing. Is that acceptable?